modules/resource-graph/query/main.bicep

```diff
@@ -0,0 +1,88 @@
+metadata name = 'Resource Graph Queries'
+metadata description = 'This module deploys a Resource Graph Query.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Optional. Name of the Resource Graph Query.')
+param name string = guid(resourceGroup().id)
```

Comment:
Is this default a good idea? Wouldn't that mean you can only deploy a single query per resource group if using the default?

Reply:
Definitely not. This is not intentional, copy/paste error, sorry about that...